types/ungap__structured-clone/json.d.ts

```diff
@@ -0,0 +1,16 @@
+export = json;
+declare namespace json {
+    /**
+     * Represent a structured clone value as string.
+     * @param {any} any some clone-able value to stringify.
+     * @returns {string} the value stringified.
+     */
+    function stringify(any: any): string;
+
+    /**
+     * Revive a previously stringified structured clone.
+     * @param {string} str previously stringified data as string.
+     * @returns {any} whatever was previously stringified as clone.
+     */
+    function parse(str: string): any;
+}
```

Comment:
```suggestion
/**
 * Represent a structured clone value as string.
 * @param {any} any some clone-able value to stringify.
 * @returns {string} the value stringified.
 */
export function stringify(any: any): string;

/**
 * Revive a previously stringified structured clone.
 * @param {string} str previously stringified data as string.
 * @returns {any} whatever was previously stringified as clone.
 */
export function parse(str: string): any;
```
(outcome is same IMO, there is no other types exported requiring namespace in module)

Reply:
I will apply this when I am on kbd next. Thank you so very much.